gsms.py

```diff
@@ -35,6 +35,9 @@
 --no_sleep
     If set, the script will not pause for 10 seconds at the end of the import.
 
+--nv_add_autodetect, -n
+    If set, the swcript will add all streamable apps from NVidia GFE's autodetected applications
```

Comment:
A nitpick, please but add a period. Same in docstrings.

Also a typo in the word `script`.

Please add this to the readme file as well.

Reply:
Done